packages/mdc-ripple/adapter.js

```diff
@@ -38,6 +38,7 @@
  *
  * @record
  */
+
```

Comment:
How'd this get into this PR? Is this fixing anything?

Reply:
hmm weird...don't remember that